### ISSUE TITLE
error logs attempt to index field 'http_host' (a nil value) 

### ISSUE THREAD

**Issue by txgo** [Mashape/kong#654]
ubuntu 14.04
kong： 0.5.2
cassandra: 2.1.11

Kong log file there are lot of error related to "Missing Host": 
```
2015/10/23 19:45:38 [error] 18832#0: *3244 [lua] app.lua:113: handle_error(): /usr/local/share/lua/5.1/lapis/nginx.lua:104: attempt to index field 'http_host' (a nil value)

stack traceback:
	/usr/local/share/lua/5.1/lapis/nginx.lua:104: in function 'fn'
	/usr/local/share/lua/5.1/lapis/nginx.lua:133: in function '__index'
	/usr/local/share/lua/5.1/lapis/application.lua:402: in function </usr/local/share/lua/5.1/lapis/application.lua:400>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/lapis/application.lua:400: in function 'dispatch'
	/usr/local/share/lua/5.1/lapis/nginx.lua:181: in function 'serve'
	content_by_lua(nginx.conf:138):9: in function <content_by_lua(nginx.conf:138):1>, client: 10.63.23.52, server: , request: "GET /status HTTP/1.0"
2015/10/23 19:45:41 [error] 18832#0: *3245 [lua] app.lua:113: handle_error(): /usr/local/share/lua/5.1/lapis/nginx.lua:104: attempt to index field 'http_host' (a nil value)

stack traceback:
	/usr/local/share/lua/5.1/lapis/nginx.lua:104: in function 'fn'
	/usr/local/share/lua/5.1/lapis/nginx.lua:133: in function '__index'
	/usr/local/share/lua/5.1/lapis/application.lua:402: in function </usr/local/share/lua/5.1/lapis/application.lua:400>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/lapis/application.lua:400: in function 'dispatch'
	/usr/local/share/lua/5.1/lapis/nginx.lua:181: in function 'serve'
	content_by_lua(nginx.conf:138):9: in function <content_by_lua(nginx.conf:138):1>, client: 10.63.23.52, server: , request: "GET /status HTTP/1.0"
```

**Comment by thibaultCha** [comment]
Apparently a Lapis issue: leafo/lapis/issues#290

**Comment by thefosk** [comment]
Fixed with 9c0db54c3172689a2edb60b019fab80c51633c5e. Will be available in the next version.
